README.md

```diff
@@ -1,67 +1,80 @@
-<!-- THIS IS AN AUTO-GENERATED FILE by ./tools/gendoc.sh. DO NOT EDIT MANUALLY -->
+<!-- THIS IS AN AUTO-GENERATED FILE by ./scripts/gendoc.sh. DO NOT EDIT MANUALLY -->
 
 # Security Intents for Intent Driven Security
+
 ![CI status](https://github.com/5GSEC/security-intents/actions/workflows/ci-verify.yml/badge.svg)
 
 Repository to hold Security Intents in standard template format.
 
 ![](res/nimbus.png)
 
-## Security Intents
-  - [blockAsset](intents/core/blockAsset.yaml)
-  - [blockRawSocket](intents/core/blockRawSocket.yaml)
-  - [protectAsset](intents/core/protectAsset.yaml)
-  - [protectPort](intents/core/protectPort.yaml)
-
 ## Security Threat Template
-```
-title:        # Scenario-title
-description:  # Detailed description of the scenario
-severity:     # High/Medium/Low
-tags: [oran, 5gcore, generic, oranThreatId]
-detectionMethods: #Mechanisms to detect the threat - https://fight.mitre.org/data%20sources
-   - name: applicationlog
-     tag: [mitre, ds0015]
-     description: desc
-     url: https://fight.mitre.org/data%20sources/DS0015
-   - name: myapplicationlog
-     tag: [accuknox, ax0015]
-     description: desc
-     url: https://...
-mitigationMethods: #Mechanisms to mitigate the threat - https://fight.mitre.org/mitigations
-   - name: applicationlog
-     tag: [mitre, ds0015]
-     description: desc
-     url: https://fight.mitre.org/data%20sources/DS0015
-   - name: myapplicationlog
-     tag: [accuknox, ax0015]
-     description: desc
-     url: https://...
+```yaml
+title: Scenario-title
+description: Detailed description of the scenario
+severity: High/Medium/Low
+tags:
+  - oran
+  - 5gcore
+  - generic
+detectionMethods: # Mechanisms to detect the threat
+  - name: Application log
+    tag:
+      - mitre/ds0015
+    description: Description # Optional
+    url: https://...
+  - name: Process
+    tag:
+      - accuknox/ax0015
+    description: Description # Optional
+    url: https://...
+mitigationMethods: # Mechanisms to mitigate the threat
+  - name: Update Software
+    tag:
+      - mitre/ds0015
+    description: desc
+    url: https://...
+  - name: User Account Management
+    tag:
+      - accuknox/ax0015
+    description: desc
+    url: https://...
 securityIntents:
-  - sample-si-1   # no params for now
-  - sample-si-2
-securityIntentBinding: sample-si-binding.yaml
-preDeploymentConsiderations: #Anything that can be done in CI/CD pipelines that can alleviate this threat
+  - sample-si-1.yaml
```

Comment:
This can be made a path.. tactics/..

Reply:
I understand your suggestion, but we can address this later based on how things progress.